notificationClient/notificationClient.go

```diff
@@ -0,0 +1,38 @@
+package notificationClient
+
+import (
+	"context"
+	"net/http"
+
+	e3dbClients "github.com/tozny/e3db-clients-go"
+	"github.com/tozny/e3db-clients-go/authClient"
+)
+
+type ToznyNotificationClient struct {
+	Host       string
+	authClient authClient.E3dbAuthClient
+}
+
+const (
+	// NotificationServiceBasePath is the base path used for notification-service calls
+	NotificationServiceBasePath = "v1/notification"
```

Comment:
take a look at the other clients, I think we're prefixing all these base paths with '/' with the thought that you should be able to join together any combination of hosts + {path1...pathN) without having to think about whether you need to add a trailing slash to the path.

Reply:
This was copied from Billing. I've changed it here, shall I change it there as well? (Not sure if other things use the BillingServiceBasePath variable since it's exposed)

Reply:
Yes please, looks like its not used by anything outside of e3db-clients-go https://github.com/search?l=&q=BillingServiceBasePath+user%3Atozny&type=Code